### PR TITLE
MINOR: Fix the secure HDFS test

### PR DESCRIPTION
**Pull request by rayokota**
This test was broken and not running because the file did not end with
*Test.java

### CODE REVIEW

**Comment by rayokota** [comment]
retest this please

**Comment by rayokota** [comment]
retest this please

File: src/test/java/io/confluent/connect/hdfs/TestWithSecureMiniDFSCluster.java [hunk 1]
```diff
@@ -90,16 +90,15 @@ public static void shutdownKdc() {
     FileUtil.fullyDelete(baseDir);
   }
 
-  @Before
+  //@Before
```
[the remark this answers is not in the document]
Comment (rayokota) [reply src/test/java/io/confluent/connect/hdfs/TestWithSecureMiniDFSCluster.java]: Tests seem to call `setUp()` directly now, like here:  https://github.com/confluentinc/kafka-connect-hdfs/blob/master/src/test/java/io/confluent/connect/hdfs/HdfsSinkTaskTestWithSecureHDFS.java#L42

The above change is parallel to the one for non-secure tests here:  https://github.com/confluentinc/kafka-connect-hdfs/blob/master/src/test/java/io/confluent/connect/hdfs/TestWithMiniDFSCluster.java#L73

**Comment by rayokota** [comment]
@rhauch , looking at the test failures, probably the new test is not cleaning up after itself properly

File: src/test/java/io/confluent/connect/hdfs/TestWithSecureMiniDFSCluster.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (wicknicks) [reply src/test/java/io/confluent/connect/hdfs/TestWithSecureMiniDFSCluster.java]: nit: delete the annotation instead of commenting it?

**Comment by wicknicks** [comment]
should the target version be `3.3.x`? this test has been around for a while.

**Comment by rayokota** [comment]
Another PR fixed the broken unit tests, which were unrelated to this PR:  https://github.com/confluentinc/kafka-connect-hdfs/pull/338

**Pull request merged**